### PR TITLE
Add Contact Form

### PR DESCRIPTION
**Pull request by KanoCode**
In this pull request, We managed the implementation of the following tasks:

- Add contact form to portfolio website. 
- Add linters to .github folder.
- Style form according to figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,6 +210,27 @@ <h1>About Myself</h1>
...
+      <label for="name"></label>
+      <input type="text" placeholder="Yeremias" id="name" maxlength="30" required>
+
+    <label for="mail"></label>
+      <input type="email" id="mail" placeholder="natajayanj14@gmail.com" required>
+  
+      <label for="message"></label>
+      <textarea name="user_msg" id="message" maxlength="500" placeholder="Write your message here" required></textarea>
+      <button type="submit">Get It Touch</button>
+    </form>
```
Comment (ShoiraTa) [inline index.html:231]: - [ ] I would kindly suggest to follow the design provided in figma: 

Figma  | Yours
------------- | -------------
image  | image

more specifically:
- please delete the border from input fields and the button,
- please add paddings inside the input fields,

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far. 

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MdSaib) [reply index.html:231]: @ShoiraTa 
Good afternoon.

After your feedback I managed to do following things:
- remove border from button & input.
- add padding to the input.

May I submit for second review now?

Comment (brytebee) [reply index.html:231]: Great job implementing the suggested changes by @ShoiraTa

**Review by brytebee — APPROVED** [review]
Hi, @MdSaib 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



## OPTIONAL

- [ ] _**Kindly include a live demo link to your README**_

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**